cmd/exec.go

```diff
@@ -58,22 +70,109 @@ var nmap = &cobra.Command{
 	Use:   "nmap",
 	Short: "execute nmap",
 	Long:  `executes nmap and splits up the job between all of the specified hosts returning the xml files to the specified directory`,
+	Args: func(cmd *cobra.Command, args []string) error {
+		_, err := deployer.ValidatePorts(nmapPorts)
```

Comment:
need to check whether the ports are within a range of legit ports

Reply:
Fixed